reference/manifest/getstarted.md

```diff
@@ -21,3 +21,9 @@ Required. The description / body content for the callout. The **resid** attribut
 Required. The URL to a page where the user can learn more about your add-in. The **resid** attribute references a valid ID in the [Urls](./resources.md#urls) element in the [Resources](./resources.md) section.
 
 > **NOTE:** **LearnMoreUrl** does not currently render in Word, Excel, or PowerPoint clients. We recommend that you add this URL for all clients so that the URL will render when it becomes available. 
+
+## Code samples
+The following are code samples that implement this feature:
```

Comment:
Let's change the intro sentence to "The following code samples use the **GetStarted** element:".